app.yaml

```diff
@@ -1,4 +1,4 @@
-application: selectives-web
+application: selectives-web-sarah
```

Comment:
I'll just reset it. I don't understand branching enough, because then do I
merge the other stuff and not this file? Maybe for a bigger change, I'll
ask you to explain it.

On Sun, Aug 30, 2015 at 5:07 PM, arthurw9 <notifications@github.com> wrote:

> In app.yaml
> <https://github.com/arthurw9/selectives-web/pull/1#discussion_r38278082>:
>
> > @@ -1,4 +1,4 @@
> > -application: selectives-web
> > +application: selectives-web-sarah
>
> I wonder if you could put this change into a different branch so you could
> have it but I don't have to pull it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/arthurw9/selectives-web/pull/1/files#r38278082>.
>